WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductCreationTypeCommand.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import UIKit
+
+/// Type to allow merchant to create a template product.
+///
+public enum ProductCreationType: Equatable {
+    case template
+    case manual
+
+    /// Title shown on the action sheet.
+    ///
+    var actionSheetTitle: String {
+        switch self {
+        case .template:
+            return NSLocalizedString("Start with a template", comment: "Title for the option to create a template product")
+        case .manual:
+            return NSLocalizedString("Add manually", comment: "Title for the option to create product manually")
+        }
+    }
+
+    /// Description shown on the action sheet.
+    ///
+    var actionSheetDescription: String {
+        switch self {
+        case .template:
+            return NSLocalizedString("Use a template to create physical, virtual, and variable products.",
```

Comment:
Are we only planning to show the physical, virtual, and variable types for templates? (Core only shows these three types, but it should be possible to use templates for all of the types in our type selector.)

If we're planning to show all the available product types, we should probably adjust this string to reflect that. Maybe something simple like:

```suggestion
            return NSLocalizedString("Use a template to create a new product.",
```

Reply:
I was planning to only show physical, virtual, and variable products to reduce the cognitive load from the new merchant. I will p2 it in a bit.

Also we can always update the copy later to match the p2 outcome.